backend/assessments/1/src/main/java/org/kdu/Main.java

```diff
@@ -0,0 +1,182 @@
+                ls.logInfo("Player Loaded" + player);
+                count++;
+            }
+
+        } catch (IOException | CsvException e) {
+            ls.logInfo(e.getMessage());
+        }
+        ls.logInfo(count +" ");
+        ls.logInfo("Teams : " + teams.size());
+    }
+    public static void main(String[] args) {
+
+        loadCsv("/home/hp/code/backend/java fundarmentals/Assesment1/src/main/resources/IPL_2021-data.csv");
+
+        ls.logInfo("Available Tasks : \n 1. Given the team's name, return all the bowlers who have taken at least 40\n" +
+                "wickets.\n" +
+                "2. Given a team display the details of the highest wicket-taker and highest\n" +
+                "run-scorer\n" +
+                "3. Fetch the top 3 run-scorer and top 3 wicket-takers of the season.");
+        int choice = sc.nextInt();
+        sc.nextLine();
+        String teamName;
+        do{
+
+            switch (choice){
+                case 1 -> {
+                    ls.logInfo("Give a team name : ");
+                    teamName = sc.nextLine();
+                    if(teams.get(teamName) == null){
+                        ls.logWarn("no team found");
+                        return;
+                    }
+                    ls.logInfo("Results : Bowlers who have taken at least 40 for" + teamName);
+                    teams.get(teamName).getBowlersbyTeam().forEach(player -> ls.logInfo(player.toString() + '\n'));
+                }
+                case 2 ->{
+                    ls.logInfo("Give a team name : ");
+                    teamName = sc.nextLine();
+                    if(teams.get(teamName) == null){
+                        ls.logWarn("no team found");
+                    }
+                    ls.logInfo("Results : Highest run scores of " + teamName);
+                    teams.get(teamName).getHighestRunScorer().forEach(player -> ls.logInfo(player.toString() + '\n'));
+                    ls.logInfo("Results : top wickets takers of " + teamName);
+                    teams.get(teamName).getHighestWicketTaker().forEach(player -> ls.logInfo(player.toString() + '\n'));
+                }
+                case 3 -> getTop3s();
+                case 4 -> {
+                    List<String[]> list = getFixture();
+                    ls.logInfo(list.size()+ "");
+//                    list.forEach(strings -> Arrays.stream(strings).forEach(ls::logInfo));
+                    try {
+                        writeToCSV(list);
+                    } catch (IOException e) {
+                        ls.logWarn("Didnt able to write in csv");
+                        return;
+                    }
+                    ls.logInfo("written to csv");
+                    return;
+                }
+
+                case 5 ->ls.logInfo("Exiting Crypto Menu. Goodbye!");
+                default -> ls.logWarn("Not a valid task");
+            }
+        }while (choice != 5);
+    }
+
+    public static void getTop3s(){
```

Comment:
For basic modularity atleast main class should only be responsible for calling appropriate classes or taking input from user at most. Make sure to make solution more modular